requirements.txt

```diff
@@ -2,9 +2,15 @@ requests
 pymongo
 together
 streamlit==1.24.1
-openai==0.27.8
+openai==1.31.1
```

Comment:
nice! haha

Reply:
Yes! 
Found that the way we were doing it was deprecated now, so using the [latest API documentation](https://platform.openai.com/docs/api-reference/completions/create?lang=python) found here for creating the chat completion.